package.json

```
@@ -92,5 +98,16 @@
       }
     },
     "specs": ""
+  },
+  "dependencies": {
+    "@react-native-community/bob": "^0.3.4"
+  },
+  "@react-native-community/bob": {
+    "source": "js",
+    "output": "web",
```

Same comment as above with the output folder name.

```suggestion
    "output": "lib",
```